Debug App/Sources/View Controllers/MerchantSessionAndSettingsViewController.swift

```diff
@@ -452,7 +452,7 @@ class MerchantSessionAndSettingsViewController: UIViewController {
     }
     
     func configureClientSession() {
-        clientSession.currencyCode = Currency(rawValue: currencyTextField.text ?? "")
+        clientSession.currencyCode = Currency("EUR")
```

Comment:
Not sure I understand. The Currency is being used outside of SDK only when configuring the ClientSession for the DebugApp, normally, the backend is doing that. We should never have this use case practically.

Reply:
Previously it got the currency from a text field in root view controller, so that you can modify what's in the session from the GUI.

This change means that you can no longer configure it, as it will always be `"EUR"`

Reply:
Ah, sorry, you are 100% right. Now I understand. Stupid mistake. 🦅